docs/content/get-started/set-up.md

```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
 <br/>
 
 
-## Clone the repos
+## Get the GDK and Starter Project Source Code
+
+### Zip Download
+
+If you are unfamiliar with Git, you can download a Zip file of the GDK and its Starter Project [here](https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3). 
+
+Once you have done that, you can skip the rest of this page and move on to [Open the FPS Starter Project]({{urlRoot}}/content/get-started/open-project.md).
+
+<%(Callout type="alert" message="
+If you can, please use Git to get the GDK and its starter project, as version control will make it easier for you to get updates in the future.")%>
+
+### Cloning the Repos using Git
 
 You need to clone two repositories; the FPS Starter Project and the GDK for Unity.
```

Comment:
maybe call out again here that if they have already downloaded the zip, they do not need to clone the repos as well